### PR TITLE
fix: update example docs to match tests

### LINKED ISSUE #804: Update HTML examples to conform to scaffolded tests
The HTML that gets deployed to `example.cypress.io` for, e.g., commands, should align with the tests that get scaffolded from `cypress/*`

### PR DESCRIPTION
**Pull request by rubysolo**
Fixes #804

### CODE REVIEW

File: app/commands/misc.html [hunk 1]
```diff
@@ -78,7 +78,7 @@ <h4 id="end"><a href="https://on.cypress.io/end">.end()</a></h4>
 // and force Cypress to re-query from the root element
 cy.get('.misc-table').within(() => {
   // ends the current chain and yields null
-  cy.contains('Cheryl').click().end()
```
Comment (MikeMcC399) [inline app/commands/misc.html:81]: Removing `.end()` makes this example no longer meaningful.

File: cypress/e2e/2-advanced-examples/misc.cy.js [hunk 2]
```diff
@@ -13,7 +13,6 @@ context('Misc', () => {
     cy.get('.misc-table').within(() => {
       // ends the current chain and yields null
       cy.contains('Cheryl').click()
-      //.end()
```
Comment (MikeMcC399) [inline cypress/e2e/2-advanced-examples/misc.cy.js:16]: same here

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
This PR needs to use semantic commit
`fix:` not `chore:` in order to generate and release a new version for upstream https://github.com/cypress-io/cypress/tree/develop/packages/example to deploy the updated app contents to https://example.cypress.io/

**Comment by MikeMcC399** [comment]
- This PR is also suffering from the issue https://github.com/cypress-io/cypress/issues/29172 causing CircleCI tests to fail on Firefox.

**Comment by MikeMcC399** [comment]
@rubysolo 

- PR https://github.com/cypress-io/cypress-example-kitchensink/pull/813 needs to be merged to get the Firefox tests to pass.
- I also made a suggestion in https://github.com/cypress-io/cypress-documentation/issues/5752#issuecomment-2010182850 about possibly using `log.end()` as a replacement example. There hasn't been any feedback on that yet.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
`end` has not been removed everywhere:

image

https://github.com/rubysolo/cypress-example-kitchensink/blob/9c9ffeb72097b578d699d92a58d2964d5aeb33b9/app/index.html#L179
```html
              <li><a href="/commands/misc">end</a></li>
```

**Review by jennifer-shehane — APPROVED** [review]

**Review by MikeMcC399 — APPROVED** [review]
LGTM! 
Good work @rubysolo !

**Pull request merged**